app/views/storage.html

```diff
@@ -30,11 +30,11 @@
             <div class="col-md-12">
               <div class="section-header page-header-bleed-right page-header-bleed-left">
                 <div class="hidden-xs pull-right" ng-if="project && ('persistentvolumeclaims' | canI : 'create')">
-                  <a ng-href="project/{{project.metadata.name}}/create-pvc" class="btn btn-default">Create Storage</a>
+                  <a ng-if="!outOfClaims" ng-href="project/{{project.metadata.name}}/create-pvc" class="btn btn-default">Create Storage</a>
```

Comment:
this situation might justify a disabled button over a missing action. thoughts @spadgett ?  If we hide the buttons then we need to take that into account in the if check when considering whether the dropdown button (kebab at mobile) should be visible.  Meaning if this individual action was hidden because of the outOfClaims check, resulting in an empty dropdown, we would want to make sure the dropdown button is hidden.